ofs/apps/cpeng/readme.md

```diff
@@ -0,0 +1,88 @@
+# hps #
+
+## SYNOPSIS ##
+
+`hps OPTIONS SUBCOMMAND SUBCOMMAND_OPTIONS`
```

Comment:
I'm not sure a base command name of hps is the best choice.  I worry there might be some conflict with something on https://rocketboards.org/.  I also think that HPS has a particular meaning to a particular SOC implementation, and we might want to think about more generic name.